src/Object/Api/Twitter/User.php

```diff
@@ -117,12 +117,12 @@ class User extends BaseDataTransferObject
 	 * @param array $publicContact         Full contact table record with uid = 0
 	 * @param array $apcontact             Optional full apcontact table record
 	 * @param array $userContact           Optional full contact table record with uid != 0
-	 * @param null  $status
+	 * @param array $status
```

Comment:
This should still be `?Status`. There's only one spot this constructor is called and the `$status` parameter is filled from `Factory\Status->createFromUriId` which returns a `Status`.
```suggestion
	 * @param ?Status $status
```

Reply:
> This should still be `?Status`. There's only one spot this constructor is called and the `$status` parameter is filled from `Factory\Status->createFromUriId` which returns a `Status`.

The __constuct() is called with an array in `src/Factory/Api/Twitter/User.php::60`. Please note the `->toArray()` at the end:

`$status = $this->status->createFromUriId($post['uri-id'], $uid)->toArray();`

Reply:
Good catch! However `Status` already is a data transfer object so it doesn't need to be converted to an array. I would like to remove the `->toArray()` call but maybe @annando can remember if and why he had to use this seemingly-superfluous call?

Reply:
Ping @annando

Reply:
We have some tests for the APIs since they are mostly decoupled from static dependencies, would you be able to test the change I suggested while we wait for @annando ?

Reply:
Short answer: I can't remember why I coded it that way.

Reply:
Ok, thanks.
@MrPetovan I changed it to provide the Status object instead of the array.